src/tokens/solana.tokenlist.json

```diff
@@ -46671,7 +46671,17 @@
         "social-token"
       ]
     }
-  ],
+    {
+      "chainId": 101,
+      "address": "7hbS1Duxm5CiVaLGZkFaqMZGcrZka5p2zX5ruYxqQgvP",
+      "symbol": "DOHA",
+      "name": "Beautiful Doha City",
+      "decimals": 0,
+      "logoURI": "https://github.com/samurtuza1/DOHA/blob/main/Doha.png",
+      "tags": [
+        "social-token"
+      ]
+  ]
```

Comment:
if it boots you again, i think this line (46684 in this example) needs to ends with a comma